vizro-core/docs/pages/user-guides/selectors.md

```diff
@@ -26,20 +26,106 @@ For more details, kindly refer to the documentation of the underlying dash compo
 
-- [dcc.Slider](https://dash.plotly.com/dash-core-components/slider])
-- [dcc.RangeSlider](https://dash.plotly.com/dash-core-components/rangeslider])
+- [`dcc.Slider`](https://dash.plotly.com/dash-core-components/slider])
+- [`dcc.RangeSlider`](https://dash.plotly.com/dash-core-components/rangeslider])
 
-???+ note
+!!! note
 
     When configuring the [`Slider`][vizro.models.Slider] and the [`RangeSlider`][vizro.models.RangeSlider] with float values, and using `step` with an integer value, you may notice
     unexpected behavior, such as the drag value being outside its indicated marks.
     To our knowledge, this is a current bug in the underlying [`dcc.Slider`](https://dash.plotly.com/dash-core-components/slider) and
     [`dcc.RangeSlider`](https://dash.plotly.com/dash-core-components/rangeslider) component, which you can circumvent by adapting the `step` size accordingly.
 
-To enhance existing selectors, please see our How-to-guide on creating [custom components](custom-components.md)
+To utilize `numerical` [`Filter`][vizro.models.Filter] selectors, the filtered column must be of `numeric` dtype,
+indicating that [pandas.api.types.is_numeric_dtype()](https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_numeric_dtype.html) must return `True` for the filtered column.
+
+## Temporal Selectors
+
+For more details, refer to the documentation of the underlying Dash components:
+
+- [`dmc.DateRangePicker`](https://www.dash-mantine-components.com/components/datepicker#daterangepicker)
+- [`dmc.DatePicker`](https://www.dash-mantine-components.com/components/datepicker)
+
+!!! note
+
+    [`DatePicker`][vizro.models.DatePicker] component is based on two underlying components `dmc.DatePicker` and `dmc.DateRangePicker`.
+    When [`DatePicker`][vizro.models.DatePicker] is configured with `range=True` (the default), the underlying component is `dmc.DateRangePicker`; when `range=False` the component is `dmc.DatePicker`.
+    When configuring the [`DatePicker`][vizro.models.DatePicker] make sure to provide your dates for `min`, `max` and `value` arguments in `"yyyy-mm-dd"` format or as datetime type.
+
+To utilize `temporal` [`Filter`][vizro.models.Filter] selectors, the filtered column must be of `datetime` dtype,
+indicating that [pandas.api.types.is_datetime64_any_dtype()](https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_datetime64_any_dtype.html) must return `True` for the filtered column.
+
+!!! note
+
+    Dataframe column type can be changed to datetime using:
+     - [pandas.to_datetime()](https://pandas.pydata.org/docs/reference/api/pandas.to_datetime.html) or
+     - [pandas.DataFrame.astype()](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.astype.html) with `datetime64` as the argument.
+
+Below is an example where all selectors are automatically set by default based on the type of the filtered column.
+
+!!! example "Default Filter selectors"
```

Comment:
Great idea of an example! I would use a well known DF though, so the yaml version is easier repeatable?

Reply:
There is no well known DF [here](https://plotly.com/python-api-reference/generated/plotly.express.data.html) that contains fine temporal, categorical and numerical columns together.

The closest we have is `px.data.gapminder(datetimes=True)` which we avoid using (https://github.com/mckinsey/vizro/pull/319#discussion_r1525060748) in the examples since datetimes are around years with `Jan-01`. So to represent DatePicker selector we use `px.data.stocks` DF in the examples (e.g. features examples).

Reply:
And in stocks, we are missing the categorical column right?

Reply:
You can transform the stocks data set by changing it from wide to long, so the columns 'GOOG', 'AAPL', 'AMZN', 'FB', 'NFLX', 'MSFT' become categories in a column we call "Stocks" and then you have a numeric column with the values :)

Then you have a data set with date, categorical and numeric values.